assembly/std/context.ts

```diff
@@ -37,7 +37,7 @@ export function ownedAddresses(): Array<Address> {
  * You get all previously called since the address of
  * the originator (transaction creator).
  *
- * @return {Array<Address>}
+ * @return {Array<Address>} returns the full call stack as a list, bottom to top
```

Comment:
this bottom to top is unclear to me. What is top ? What is bottom ?

I propose:
```typescript
/*
 * Returns the stack of called addresses.
 * 
 * The first element of this list is the originator (creator of the transaction) and the last is the address of the current smart-contract.
 *
 * When executing the ExecuteSC function, the list is composed of only one entry: the initiator, the caller and the callee having with the same address.
 * 
 * @return {Array<Address>}
```